www/gatsby-node.js

```diff
@@ -15,6 +15,14 @@ const packages = require(`./src/utils/node/packages.js`)
 const features = require(`./src/utils/node/features.js`)
 const sections = [docs, showcase, starters, creators, packages, features]
 
+exports.createSchemaCustomization = async helpers => {
```

Comment:
This is super janky (in general, as a pattern) but aware there's probably reasons for grouping into sections.